homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -851,15 +848,10 @@ async def async_setup_entry(
         entities.append(
             PlenticoreDataSensor(
                 process_data_update_coordinator,
+                description,
                 entry.entry_id,
                 entry.title,
-                module_id,
-                data_id,
-                name,
-                sensor_data,
-                PlenticoreDataFormatter.get_method(fmt),
                 plenticore.device_info,
-                None,
```

Comment:
Note: I had planned this for a follow-up PR but it leaked into this one...
The entity_category was always passed as None so I simply removed it.